packages/appium/lib/cli/extension-command.js

```diff
@@ -486,4 +487,51 @@ export default class ExtensionCommand {
     this.config.installedExtensions[ext].version = version;
     await this.config.write();
   }
-}
+
+  /**
+   * Runs a script cached inside the "scripts" field under "appium"
+   * inside of the driver/plugins "package.json" file
+   *
+   * @param {string} ext - name of the extension to run a script from
+   * @param {string} scriptName - name of the script to run
```

Comment:
missing the description of the returned result and of possible thrown exceptions

Reply:
still not resolved. Is the method supposed to return a string a an array of strings?

Reply:
Seems like I must've messed up during git rebase, let me try again these comments should have been resolved